src/pages/projects.tsx

```diff
@@ -20,9 +43,84 @@ export default function Home() {
 			</Head>
 
 			<Layout pageName="PROJECTS">
-				{/* <Project /> */}
+				<img src="https://i.imgur.com/tzzAJeX.png" className={styles.background_img} />
```

Comment:
need a **local image** for this

image optimization will be disabled if using external links :smile: